### ISSUE TITLE
Add symfony 4 compatibility

### ISSUE THREAD

**Issue by pyrech** [jolicode/JoliTypoBundle#2]
Also, we could IMO move this bundle as an optional bridge in the original lib to ease maintenance.

**Comment by damienalexandre** [comment]
Yes, this bundle could be deleted/deprecated in favor of a bridge in the library itself, only if this can be light (only 2/3 files for Symfony). Because JoliTypo is used in a lot of non symfony projects.

**Comment by pyrech** [comment]
Yep, I think there will be only the DIC part (Configuration and Extension) and the (empty) bundle class.
The Twig extension can be moved as a Twig bridge.
And the Templating helper can be... just removed IMO :stuck_out_tongue:
